BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -237,6 +284,103 @@ extension StoresTableViewController {
     
 }
 
+//MARK: Mail Code
+extension StoresTableViewController: MFMailComposeViewControllerDelegate {
+    
+    fileprivate func sendEmail() {
+        
+        if !MFMailComposeViewController.canSendMail() {
+            
+            return
+            
+        }
+        else {
+            
+            self.present(configureMailComposeViewController(), animated: true, completion: nil)
+            
+        }
+        
+    }
+    
+    fileprivate func checkUserEmailSetting(in noResultsCell: NoResultsCustomCell) {
```

Comment:
Name of the function is misleading. Titled how it is right now, I expect it to return a `Bool` telling me whether it is available or not.

What could be a better name for this?